.eslintrc.yml

```diff
@@ -18,7 +18,7 @@ parserOptions:
 plugins:
   - '@typescript-eslint'
 rules:
-  '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_' }]
+  '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }]
```

Comment:
If there's a reason that this convention wasn't being ignored before, let me know.